source/features/linkify-user-location.tsx

```diff
@@ -12,10 +12,17 @@ function addLocation(baseElement: HTMLElement): void {
 		}
 
 		const locationName = location.textContent!.trim();
-		const googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(locationName)}`;
+		const googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${locationName}`;
```

Comment:
Why not? What if their location is `nah&somethingelse=do`?

Reply:
Good call, reverted